service/controller/clusterapi/v29/controllercontext/status.go

```diff
@@ -71,6 +71,7 @@ type ContextStatusTenantClusterTCCP struct {
 	IsTransitioning   bool
 	MachineDeployment v1alpha1.MachineDeployment
 	RouteTables       []*ec2.RouteTable
+	SecurityGroups    ContextStatusTenantClusterTCCPSecurityGroups
```

Comment:
IMO plural only when we have a list. This is a dictionary and singular reads well for the caller. 

```suggestion
	SecurityGroup    ContextStatusTenantClusterTCCPSecurityGroup
```